Challenge/App.tsx

```diff
@@ -0,0 +1,81 @@
+
+import React from 'react';
+import {
+  SafeAreaView,
+  ScrollView,
+  StatusBar,
+  StyleSheet,
+  Text,
+  View,
+  Pressable,
+  Image,
+} from 'react-native';
+import avatar from './src/components/img/avatar.png';
+import ListOfCurrencies from './src/components/ListOfCrypto';
+
+
+const App = () => {
+
+  const addCurrencyHandler = () =>{
+      console.log("Estás tocando")
```

Comment:
Delete this log and the function if it doesn't do anything. And leave only one blank space before the return